app/views/reports/regions/_facility_tables.html.erb

```diff
@@ -1,277 +0,0 @@
-<h3 class="mt-48px mb-8px fs-28px c-black">
```

Comment:
These 4 tables were displayed in the facilities page at the bottom earlier, we don't show them anymore.